platform/harmony/client.go

```diff
@@ -0,0 +1,44 @@
+package harmony
+
+import (
+	"fmt"
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/blockatlas/util"
+	"strconv"
+)
+
+type Client struct {
+	blockatlas.Request
+}
+
+func (c *Client) GetTxsOfAddress(address string) (txPage *TxResult, err error) {
+	params := []interface{}{
+		map[string] interface{}{"address": address, "fullTx": true},
+	}
+	err = c.RpcCall(&txPage, "hmy_getTransactionsHistory", params)
```

Comment:
Use struct
```
	type params struct {
		address string `json:"address"`
		fullTx  bool       `json:"fullTx"`
	}
	err = c.RpcCall(&txPage, "hmy_getTransactionsHistory",  params{address: address, fullTx: true})
```

Reply:
@Pantani tried to used structs and for some reason the `fullTx` boolean gets translated as `"true"` and not `true`. Working with a map inside an interface makes it work properly